front/src/modules/powerRestriction/helpers/getRestrictionsToResize.ts

```diff
@@ -8,96 +10,133 @@ import { NO_POWER_RESTRICTION } from '../consts';
 const getPowerRestrictionFromRange = (
   pathSteps: PathStep[],
   powerRestrictionRanges: PowerRestriction[],
-  rangeData: IntervalItem
-): PowerRestriction | null => {
+  rangeData: IntervalItem | undefined
+): PowerRestriction | undefined => {
+  if (!rangeData) return undefined;
+
   const fromPathStep = getPathStep(pathSteps, rangeData.begin);
   const toPathStep = getPathStep(pathSteps, rangeData.end);
 
-  if (!fromPathStep || !toPathStep) return null;
+  if (!fromPathStep || !toPathStep) return undefined;
 
   const powerRestrictionRange = powerRestrictionRanges.find(
     (restriction) => restriction.from === fromPathStep.id && restriction.to === toPathStep.id
   );
-  return powerRestrictionRange || null;
+  return powerRestrictionRange || undefined;
```

Comment:
This should be enough I think ?
```suggestion
  return powerRestrictionRange;
```

Reply:
Done